mobile-workspace/latest/config/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configure Mobile Workspace
+---
+
+Find Alfresco Mobile Workspace at [Google Play](https://play.google.com/store/apps/details?id=com.alfresco.content.app&hl=en-GB&ah=tbrTPT50zRyrvFT0xxYx0IRH6DA){:target="_blank"} for Android or [Apple App Store](https://apps.apple.com/gb/app/alfresco-mobile-workspace/id1514434480){:target="_blank"} for iOS.
+
+## Login screen
+
+From the first screen you enter the URL of your Alfresco Content Services instance into the `Connect to`  field. You will then be asked to authenticate using your Alfresco credentials. If you want to use Single Sign-On (SSO) to connect to the Mobile Workspace you can enter the URL of your Alfresco Identity Management Service. You can enter the URL in the following format `my.alfresco.com`.
```

Comment:
`Alfresco Identity Management Service` isn't the product name that's used in the Docs site and may confuse customers. Should this be just 'Identity Service` so it matches the later reference (line 22)?

Reply:
done